manage.py

```diff
@@ -9,8 +7,64 @@
 migrate = Migrate(app, db)
 
 manager = Manager(app)
-manager.add_command('db', MigrateCommand)
 
-if __name__ == '__main__':
-    manager.run()
 
+@manager.command
+def init():
+    "Initiates the application for first time use"
+
+    from cleansweep.loaddata import init
+    init()
+
+
+@manager.command
+def load():
+    "Loads data from specified directory"
+
+    from cleansweep.loaddata import main
+    root_dir = raw_input("Directory (Input without quotations): ")
+    main(root_dir)
```

Comment:
I don't think it allows more than one argument. You yourself mentioned in the issue. I didn't see anything in documentation but I'll look.

I used classes because dash "-" in function name isn't a valid.
I'll change it to underscore.
________________________________
From: Anand Chitipothu<mailto:notifications@github.com>
Sent: ‎22/‎09/‎2015 06:59
To: AamAadmiParty/cleansweep<mailto:cleansweep@noreply.github.com>
Cc: Raghav Sharma<mailto:Raghav_ftw@hotmail.com>
Subject: Re: [cleansweep] Integrated run.py commands with flask-script and moved it to manage.py (#120)

>
> +@manager.command
> +def init():
> +    "Initiates the application for first time use"
> +
> +    from cleansweep.loaddata import init
> +    init()
> +
> +
> +@manager.command
> +def load():
> +    "Loads data from specified directory"
> +
> +    from cleansweep.loaddata import main
> +    root_dir = raw_input("Directory (Input without quotations): ")
> +    main(root_dir)

No raw_input. It should accept arguments from command-line.

---
Reply to this email directly or view it on GitHub:
https://github.com/AamAadmiParty/cleansweep/pull/120/files#r40046598